tests/unit/config/test_converter_profiling_phase_routes.py

```
@@ -383,3 +385,57 @@ def test_nested_adaptive_scale_string_false_disables_phase(self):
+        json_path.write_text(
+            '{"points":[{"time_s":0,"qps":5},{"time_s":60,"qps":10}]}',
+            encoding="utf-8",
+        )
+        loadgen = CLIConfig(
+            request_rate=100.0,
+            request_rate_series=json_path,
+            request_count=10,
+        )
+        user = _make_user(loadgen=loadgen)
+
+        with pytest.raises(ValueError, match="request-rate.*request-rate-series"):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)